pillar/apps/ocw-next-production.sls

```diff
@@ -4,9 +4,11 @@ ocw-next:
   source_data_bucket: open-learning-course-data-production
   search_api_url: //open.mit.edu/api/v0/search/
   ocw_to_hugo_git_ref: release
-  hugo_course_publisher_git_ref: release
+  ocw_www_git_ref: main
```

Comment:
Are we going to deploy QA and production from the same branch?

Reply:
Yes, but only because at this moment there is one branch. I expect this to change.

Reply:
I've changed to `release` for production in https://github.com/mitodl/salt-ops/pull/1336/commits/4a219844904f93fa3500b4136d1efba6161ecae6 and will change RC later to `release-candidate` when such a branch exists.